src/components/body/image-diff.js

```diff
@@ -0,0 +1,146 @@
+import { replaceGCSUrlOrigin } from '@twreporter/core/lib/utils/storage-url-processor'
+import mq from '../../utils/media-query'
```

Comment:
`import mq from '@twreporter/core/lib/utils/media-query'`

Reply:
Updated. Also fix the `mq` importing in the audio.